tests/test_colored_noise.py

```diff
@@ -0,0 +1,81 @@
+import random
+import unittest
+
+import torch
+
+from torch_audiomentations import AddColoredNoise
+from torch_audiomentations.utils.io import Audio
+from .utils import TEST_FIXTURES_DIR
+
+
+class TestAddBackgroundNoise(unittest.TestCase):
```

Comment:
This should ideally be renamed to TestAddColoredNoise

Reply:
Sloppiness of mine, I'll change accordingly